### ISSUE TITLE
1.21 Update

### ISSUE THREAD

**Issue by DrHeretic** [Alathra/SiegeEngines#11]
Is there going to be an update?

**Comment by ccorp2002** [comment]
Yes, I however, the server this was originally built for is not on 1.21.

Updating this should be as easy as updating the command api version the plugin uses, which I can do soon.

**Comment by DrHeretic** [comment]
Awesome thankyou!

**Comment by AevumTacet** [comment]
Any news on this? 👀

**Comment by Mineopo** [comment]
Nope

**Comment by darksaid98** [comment]
~~Try the latest release candidate [here](https://github.com/Alathra/SiegeEngines/releases/tag/v1.1.0-RC-1), [SiegeEngines-1.1.0-RC-1.jar](https://github.com/Alathra/SiegeEngines/releases/download/v1.1.0-RC-1/SiegeEngines-1.1.0-RC-1.jar).~~

Grab the latest release candidate from [the releases page here](https://github.com/Alathra/SiegeEngines/releases).

This patch should include support for `1.20.4` upwards to `1.21.3`.

**Comment by ShermansWorld** [comment]
Release created on Spigot: [https://www.spigotmc.org/resources/siegeengines.116034/updates](url)

**Comment by ShermansWorld** [comment]
Closing
